### PR TITLE
Updates for EthIO Revision

### PR DESCRIPTION
**Pull request by callendorph**
Moves the reset circuit as a reusable unit to this repo.
Increment to version 0.3.0

### CODE REVIEW

File: slm.toml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 name = "debug"
-version = "0.2.0"
+version = "0.3.0"
 [dependencies]
 jsl = { git = "JITx-Inc/jsl", version = "0.7.0" }
+switches = { git = "JITx-Inc/jsl", version = "0.1.0"}
+diodes = { git = "JITx-Inc/jsl", version = "0.2.0"}
```
Comment (bhusang) [inline slm.toml:6]: Are these the right repo addresses?

File: src/circuits/ResetSwitch.stanza [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+This creates an active low reset circuit with a manual
+switch controlled reset. This circuit would be typical for
+a prototype and is customizable.
+
+@member rail Voltage rail for generating the reset signal
+@member reset_n Active low reset signal generated from tactle switch
+
+@param resistance Pullup resistance value for the reset signal. Default is 10k.
+@param delay RC delay constant in seconds. Default is 1nS
```
Comment (bhusang) [inline src/circuits/ResetSwitch.stanza:23]: In MKS format, this ought to be `1ns`

**Review by bhusang — APPROVED** [review]

File: slm.toml (hunk 1 above)
Comment (callendorph) [reply slm.toml:6]: Nope - those were placeholders. Will fix.

File: src/circuits/ResetSwitch.stanza (hunk 2 above)
Comment (callendorph) [reply src/circuits/ResetSwitch.stanza:23]: :+1:

**Pull request merged**